clients/client-python/tests/unittests/test_gravitino_version.py

```diff
@@ -0,0 +1,90 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import unittest
+
+from gravitino.client.gravitino_version import GravitinoVersion
+from gravitino.dto.version_dto import VersionDTO
+from gravitino.exceptions.gravitino_runtime_exception import GravitinoRuntimeException
+
+
+class TestGravitinoVersion(unittest.TestCase):
```

Comment:
Can you please check if it is possible to add a python integration test for this feature?

Reply:
Actually it is automatically turned on in integration test(the default value of `check_version` is `true`), so everytime we construct `GravitinoClient` and `GravitinoAdminClient`, the version check wil run.